docs/wiki/file_manipulation.md

```diff
@@ -0,0 +1,528 @@
+```
+
+@:@
+
+## Deleting files and directories
+
+### `delete`
+
+Deletes a file or empty directory that must exist (otherwise it will fail). 
+
+@:select(api-style)
+
+@:choice(syntax)
+
+```scala mdoc:compile-only
+path.createFile >> 
+  path.write("TOP SECRET 🚫, MUST DELETE") >>
+    path.delete
+```
+
+@:choice(static)
+
+```scala mdoc:compile-only
+FilesOs.createFile(path) >>
+  FilesOs.write(path, "TOP SECRET 🚫, MUST DELETE") >>
+    FilesOs.delete(path)
+```
+
+@:choice(fs2)
+
+```scala mdoc:compile-only
+for
+  _ <- Files[IO].createFile(path)
+
+  _ <- Stream.emit("TOP SECRET 🚫, MUST DELETE")
+        .through(Files[IO].writeUtf8(path))
+        .compile
+        .drain
+
+  _ <- Files[IO].delete(path)
+yield ()
+```
+
+@:@
+
+### `deleteIfExists`
+
+Similar to `delete`, but returns `true` if the deletion was successful instead of failing: 
+
+@:select(api-style)
+
+@:choice(syntax)
+
+```scala mdoc:compile-only
+path.deleteIfExists >>= 
+  (deleted => IO.println(s"Was the file deleted? $deleted"))
+```
+
+@:choice(static)
+
+```scala mdoc:compile-only
+FilesOs.deleteIfExists(path) >>= 
+  (deleted => IO.println(s"Was the file deleted? $deleted"))
+```
+
+@:choice(fs2)
+
+```scala mdoc:compile-only
+Files[IO].deleteIfExists(path) >>= 
+  (deleted => IO.println(s"Was the file deleted? $deleted"))
+```
+
+@:@
+
+### `deleteRecursively`
+
+With the previous functions, the directory had to be empty to be deleted. The difference with this method is that it recursively deletes all files or folders contained inside it, optionally following symbolic links if specified: 
+
+Recursively deletes all files or folders, following symbolic links if specified. 
+
+Note that, unlike the previous functions, this one will not fail if the directories are empty or do not exist:
+
+@:select(api-style)
+
+@:choice(syntax)
+
+```scala 3
+val dirs = Path("this/folders/will/be/created/and/deleted")
+
+for 
+  _ <- dirs.createDirectories
+  _ <- dirs.deleteRecursively // Will delete all of them!
+yield ()
+```
+
+@:choice(static)
+
+```scala 3
+val dirs = Path("this/folders/will/be/created/and/deleted")
+
+for 
+  _ <- FilesOs.createDirectories(dirs)
+  _ <- FilesOs.deleteRecursively(dirs) // Will delete all of them!
+yield ()
+```
+
+@:choice(fs2)
+
+```scala 3
+
+val dirs = Path("this/folders/will/be/created/and/deleted")
+
+for 
+  _ <- Files[IO].createDirectories(dirs)
+  _ <- Files[IO].deleteRecursively(dirs) // Will delete all of them!
+yield ()
+```
+
+@:@
+
+
+## File operations
+
+Working directly with files is a common need in many scripting scenarios. This library provides essential functions for renaming, moving, and copying files, allowing you to efficiently manage and organise your data. 
+
+### `copy`
+
+Copies a file from a source path to a target path. The method will fail if the destination path already exists; to avoid this behaviour, you can pass flags to e.g., replace the contents in the destination:
```

Comment:
```suggestion
Copies a file from a source path to a target path. The method will fail if the destination path already exists; to avoid this behaviour, you can, for example, pass flags to replace the contents at destination:
```